### PR TITLE
Create separate ginger_embed module

### PR DESCRIPTION
**Pull request by DorienD**
Ginger embed module for embedding ginger resources on other ginger sites or not ginger sites

### CODE REVIEW

**Comment by ddeboer** [comment]
What do we gain by the separate module, apart from mod_ginger_rdf? Mod_ginger_rdf still needs to be enabled for the embeds to work.

**Comment by DorienD** [comment]
When it was made for hartamsterdam the files were all over te place, now its centralised in one module. And yes rdf still needs to be enabled. Hmmm.

**Comment by DorienD** [comment]
I could move everything into mod_ginger_rdf. Also the erlang. Should we then make a config key if you want to have the ginger embeds or not?

File: modules/mod_ginger_embed/templates/embed/share-link.tpl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<a id="{{ #embed }}" href="" title="{_ Embed _}" class="">{_ Embed _}</a>
```
Comment (ddeboer) [inline modules/mod_ginger_embed/templates/embed/share-link.tpl]: This file has the wrong path: it must be `templates/share/share-link.tpl` to be picked up by mod_ginger_base’s `share.tpl`.

**Comment by ddeboer** [comment]
>  Should we then make a config key if you want to have the ginger embeds or not?

Actually, when there’s something to toggle (enable/disable), a separate module makes sense: enable/disable the module.

In this case the only toggable is the `share-link.tpl` template, isn’t it?

File: modules/mod_ginger_embed/mod_ginger_embed.erl [hunk 2]
```diff
@@ -0,0 +1,62 @@
+%% @author Driebit <info@driebit.nl>
+%% @copyright 2015
+
+-module(mod_ginger_embed).
+-author("Driebit <info@driebit.nl>").
+
+-mod_title("Ginger Embed").
+-mod_description("Ginger Embed").
+-mod_prio(250).
+-mod_depends([mod_content_groups, mod_acl_user_groups]).
```
Comment (ddeboer) [inline modules/mod_ginger_embed/mod_ginger_embed.erl]: Add `mod_ginger_rdf`.

Comment (DorienD) [inline modules/mod_ginger_embed/mod_ginger_embed.erl]: done.

**Comment by DorienD** [comment]
I've moved the share link, so it works together with the all include from ginger base.

**Pull request merged**